packages/engine/src/tools/build-validated.ts

```
@@ -0,0 +1,137 @@
+            el.setAttribute(patch.attribute, patch.value ?? '')
+            appliedAny = true
+          } else if (patch.action === 'removeAttribute' && patch.attribute) {
+            el.removeAttribute(patch.attribute)
+            appliedAny = true
+          } else if (patch.action === 'replaceOuterHTML' && patch.value !== undefined) {
+            const newEl = parse(patch.value)
+            el.replaceWith(newEl)
+            appliedAny = true
+          }
+        }
+      } catch (err) {
```

ESLint

[error] 'err' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] Empty block statement.

(no-empty)